src/ILCompiler.ReadyToRun/src/ObjectWriter/R2RPEBuilder.cs

```diff
@@ -16,15 +16,72 @@
 using ILCompiler.DependencyAnalysis;
 using ILCompiler.DependencyAnalysis.ReadyToRun;
 
+using Internal.TypeSystem;
+
 namespace ILCompiler.PEWriter
 {
+    public static class TargetExtensions
```

Comment:
Separate class file?

Reply:
OK, moved into new file in 6th commit together with the ModuleOSOverride enumeration.